src/utils/getUserImg.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const getUserImg = (username) => {
+  return axios
+    .get(`https://project-nc-news-xdpp.onrender.com/api/users/${username}`)
+    .then(({ data: { user } }) => {
+      return user.avatar_url;
+    });

```

Comment:
When you finish the course you might want to make an endpoint for this and handle it on the backend, maybe even fetch the image URLs along with the comments but for now this is a solid solution to the problem 👍